src/constants.ts

```diff
@@ -1,6 +1,7 @@
-import type { PM } from 'detect-package-manager'
 import { colorItBold, colorIt } from './utils.js'
 
+type PM = 'npm' | 'pnpm' | 'yarn' | 'bun';
```

Comment:
This should not be needed as `PM` from `detect-package-manager` exports exactly this type: https://github.com/egoist/detect-package-manager/blob/main/src/index.ts#L5

Reply:
I removed three packages, including this as it is no longer needed and is buggy (it produced the yarn bug)

Reply:
Can you describe the bug you experience? I haven't seen anything buggy so far.

Reply:
Sure, in the issue that I linked in the description it was mentioned that yarn was sometimes used, now we had this comment:
```
        /**
         * create a package-lock.json, yarn.lock or pnpm-lock.yaml so
         * that `detect-package-manager` doesn't mark it as a default
         * Yarn project
         * @see https://github.com/egoist/detect-package-manager/issues/11
         */
```
The logic is simplified, if you use `npm init wdio .` it will use npm, if you use `yarn create wdio .` it will use yarn, if you use `pnpm create wdio .` it will use pnpm etc, we should not need a package for this as the logic is already there and it is really simple.

Reply:
Furthermore, when using for example yarn, all commands need to be run with yarn. same goes for the other package managers so this has now also been resolved.